### ISSUE TITLE
`@stylistic/object-curly-spacing`: add option to ignore ImportDeclaration

### ISSUE THREAD

**Issue by coderaiser** [eslint-stylistic/eslint-stylistic#893]
### Clear and concise description of the problem

The rule reports ImportDeclaration:

```diff
-import json from './mod.json' with { type: 'json' };
+import json from './mod.json' with {type: 'json'};
```

But that is not and object at all, and it is more visible that type is `json`.
Object is different case, since it is better to write it in couple lines:

```js
const a = {
   hello: 'world',
};
```

But couple lines for ImportDeclaration is looks like too much:

```js
import json from './mod.json' with {
    type: 'json'
};
```

### Suggested solution

Would be great to add option for such special case to ignore it, just like it was before.

### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Contributes

- [ ] If this feature request is accepted, I am willing to submit a PR to fix this issue

**Comment by 9romise** [comment]
Hi 👋,
Thanks for sharing this perspective! To better understand your use case:

I'm curious about whether this approach is what you'd prefer (with default options):

``` ts
const obj = {foo:1}

const {foo} = obj

import {name, version} from 'package.json' with { type: 'json' }

export {foo}

type Foo = {bar: string};

interface Foo {bar: string};

enum Foo {Bar};
```

Appreciate you helping us find the right balance!

**Comment by coderaiser** [comment]
I would prefer this:

```ts
const obj = {
    foo: 1,
};

const {foo} = obj;

import {name, version} from 'package.json' with { type: 'json' }

export {
   foo,
}

type Foo = {
    bar: string;
};

interface Foo {
    bar: string;
};

enum Foo {
    Bar
};
```
